docs/source/installation.rst

```diff
@@ -100,6 +100,16 @@ If you prefer a ``pip`` installation, we provide the following command
 Note that, differently from the  ``conda`` command, the above **will not** create a virtual environment.
 Make sure you create and activate your environment previously.
 
+Enable Nvidia Collective Communication Library (NCCL)
+=======================================================
+To obtain highly-optimized performance on GPU clusters, PyLops-MPI also supports the Nvidia's collective communication calls (NCCL).
```

Comment:
I think the link needs `_ at the end... I would also remove (NCCL). and put (NCCL) in the link